katas/content/multi_qubit_gates/index.md

```diff
@@ -121,6 +121,82 @@ $$\alpha|00\rangle + \beta|11\rangle$$
 
 The $CNOT$ gate is self-adjoint: applying it for the second time reverses its effect.
 
+
+@[exercise]({
+    "id": "multi_qubit_gates__two_qubit_gate_1",
+    "title": "Two Qubit Gate 1",
```

Comment:
Originally the task names looked like this because there was no theory in the BasicGates kata, so part of the challenge was figuring out which gate was supposed to be used. Now there is no mystery (the only gate we learned so far is CNOT), so we can use more descriptive names.
How about we call this task "Entangle Qubits"? (And rename the folder too)